### ISSUE TITLE
hex logo for webchem

### ISSUE THREAD

**Issue by Aariq** [ropensci/webchem#252]
The webchem package could use a logo.  The natural choice is, of course, a benzene ring, but it would be great if it indicated a little more about what webchem does.

**Comment by stitam** [comment]
How about a chemical factory icon like one if these?
https://www.shutterstock.com/image-vector/set-factory-manufacturing-icon-black-glyph-1473454799
it is not what webchem _does_, but people will probably get the idea:)

**Comment by Aariq** [comment]
I also thought of maybe incorporating a database icon, but that icon is so terribly boring. ![](https://cdn4.iconfinder.com/data/icons/computers-3/32/574-01-512.png)

**Comment by jmorim** [comment]
I whipped something up today, thoughts?

![webchem_logo](https://user-images.githubusercontent.com/34515725/81351327-0d3dcb80-9092-11ea-9bdf-10c3116566c4.png)

**Comment by andschar** [comment]
> I whipped something up today, thoughts?
> 
> ![webchem_logo](https://user-images.githubusercontent.com/34515725/81351327-0d3dcb80-9092-11ea-9bdf-10c3116566c4.png)

Looks a bit like a spider web in my opinion and a bit too symmetric for my taste.

**Comment by andschar** [comment]
I like the idea of combining something chemical, like the mentioned benzene ring and something data related. Maybe a cloud logo instead of the db logo?

**Comment by stitam** [comment]
Thanks @jmorim for joining the discussion! Even though "web" is present in the name of the package, because of ChemSpider, I think it is better to avoid any association with spiders.

**Comment by stitam** [comment]
> Maybe a cloud logo instead of the db logo?

If it's a cloud, then the chemicals in it'd better be harmless;)
or fun, like N2O:D

**Comment by jmorim** [comment]
![webchem_logo_v2](https://user-images.githubusercontent.com/34515725/81451416-7127c900-9152-11ea-9911-383f16e0bc83.png)
Looking better?

**Comment by Aariq** [comment]
Here's another idea:  BTW, I like the web in the previous one, but would love a way for "webchem" to be inside of the hexagon.
![logo](https://user-images.githubusercontent.com/25404783/81484751-27081b80-9216-11ea-84eb-07aa23375412.png)

**Comment by stitam** [comment]
I agree with webchem inside the hexagon. Love the beaker-database idea:) how about _barrels_?

* https://www.iconfinder.com/icons/2443036/barrels_cartoon_fuel_gas_oil_petrol_pump_icon
* https://www.123rf.com/photo_107311436_stock-illustration-metal-barrel-with-oil-icon-cartoon-illustration-of-metal-barrel-with-oil-icon-for-web.html
* https://www.istockphoto.com/ie/vector/toxic-waste-gm472281171-16035819

If the drawing is related to chemistry, I don't think we need the benzene ring as well.

**Comment by andschar** [comment]
I also think the "webchem" should be inside the hexagon. The beaker database is also a nice idea :)

**Comment by PythonCoderUnicorn** [comment]
i design hex logos, i made one for [chemr](https://github.com/paleolimbot/chemr) 

here are 2 designs i came up with. 

![Screen Shot 2022-12-01 at 9 47 09 AM](https://user-images.githubusercontent.com/55933131/205111025-dd24d170-411c-47a3-b281-45fa5a9f4a1c.png)

**Comment by stitam** [comment]
Love them! They show that webchem is a data retrieval package and the package is about chemicals.

**Comment by PythonCoderUnicorn** [comment]
Lovely beans, 

here is the second logo design.  I always find that positioning the logo in the README to be best as followed:

```
<img height='200' align='right' src=" <path>" </>
```


![webchem 2](https://user-images.githubusercontent.com/55933131/205117650-4850c419-b095-426e-a2af-887035f91c61.png)

Happy to help ! 💯

**Comment by dmi3kno** [comment]
We need a Github card.  I am concerned about your use of proprietary Futura font. Would you consider switching to DM Sans (or alternatively to Jost/Kumbh Sans/Red Hat Text/Questrial/Nunito)?

```r
library(magick)
#remotes::install_github("dmi3kno/bunny")
library(bunny)

wbch_hex <- image_read("~/Downloads/205117650-4850c419-b095-426e-a2af-887035f91c61.png")

fg_col <- "#344a80"
bg_col <- "white"

# this will write into documentation folder where it needs to be
wbch_hex %>%
  image_scale("200x200") %>%
  image_write("man/figures/logo.png", density = 600)

wbch_hex_gh <- wbch_hex %>%
  image_scale("400x400")


gh_logo <- bunny::github %>%
  image_scale("40x40")

# Alternative texts
# - Your chemistry companion on the web.

wbch_ghcard <- image_canvas_ghcard(fill_color = bg_col) %>%
  image_composite(wbch_hex_gh, gravity = "East", offset = "+100+0") %>%
  image_annotate("Your chemistry", gravity = "West", location = "+100-50",
                 color=fg_col, size=60, font="DM Sans", weight = 500) %>%
  image_annotate("companion in R.", gravity = "West", location = "+100+20",
                 color=fg_col, size=60, font="DM Sans", weight = 500) %>%
  image_compose(gh_logo, gravity="West", offset = "+100+100") %>%
  image_annotate("ropensci/webchem", gravity="West",
                 location="+150+100", size=45, font="Ubuntu Mono") %>%
  image_border_ghcard(bg_col) 

wbch_ghcard %>% image_scale("50%")

wbch_ghcard %>%
  image_write("data-raw/wbch_ghcard.png", density = 600)
```

![wbch_ghcard](https://user-images.githubusercontent.com/13419011/205246368-ba2f5177-1729-4c2e-a95b-3a8157e6bcc8.png)

**Comment by PythonCoderUnicorn** [comment]
the font used is Jost.

**Comment by stitam** [comment]
Thanks @dmi3kno for providing the script for generating the GH card! I'm unsure what these are for,  can you explain this a bit? An informative link would be perfect :) Is this something that would automatically show up on e.g. social media platforms when we share a link to the repo?

**Comment by dmi3kno** [comment]
[Github social preview cards](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview) allow you to use more informative image when sharing the link to the repo. You need to upload this image to the respective section in Github settings for the repo, wait a few min and the social preview card will appear when you share the link  on social media.

[Here's](https://twitter.com/dmi3k/status/1556979602303733761) how social preview card looks for my package.
